.github/workflows/build_and_test.yml

```diff
@@ -478,6 +478,7 @@ jobs:
         # Jinja2 3.0.0+ causes error when building with Sphinx.
         #   See also https://issues.apache.org/jira/browse/SPARK-35375.
         python3.9 -m pip install 'flake8==3.9.0' pydata_sphinx_theme 'mypy==0.920' numpydoc 'jinja2<3.0.0' 'black==21.5b2'
+        python3.9 -m pip install pandas-stubs
```

Comment:
just qq: Why new line instead of just adding to the existing list?

`python3.9 -m pip install 'flake8==3.9.0' pydata_sphinx_theme 'mypy==0.920' numpydoc 'jinja2<3.0.0' 'black==21.5b2' pandas-stubs`

Reply:
Oh..the above line is for build. never mind!